extension/app/ts/components/pages/AddNewAddress.tsx

```diff
@@ -78,29 +94,31 @@ export function AddNewAddress(param: AddAddressParam) {
 			<section class = 'modal-card-body' style = 'overflow: visible;'>
 				<div class = 'card' style = 'margin: 10px;'>
 					<div class = 'card-content'>
-						<div class = 'media'>
-							<div class = 'media-left'>
-								<figure class = 'image'>
-									{ addressInput && ethers.utils.isAddress(addressInput.trim()) ?
-										<Blockie seed = { addressInput.trim().toLowerCase() } size = { 8 } scale = { 5 } />
-									: <div style = 'background-color: var(--unimportant-text-color); width: 40px; height: 40px; border-radius: 5px;'/> }
-								</figure>
-							</div>
+						{ addressType !== 'addressInfo' ? <p class = 'paragraph'> { `No support to rename this address type yet :( (${ addressType })` } </p> :
+							<div class = 'media'>
+								<div class = 'media-left'>
+									<figure class = 'image'>
+										{ addressInput && ethers.utils.isAddress(addressInput.trim()) ?
+											<Blockie seed = { addressInput.trim().toLowerCase() } size = { 8 } scale = { 5 } />
+										: <div style = 'background-color: var(--unimportant-text-color); width: 40px; height: 40px; border-radius: 5px;'/> }
+									</figure>
+								</div>
 
-							<div class = 'media-content' style = 'overflow-y: unset; overflow-x: unset;'>
-								<input className = 'input' type = 'text' value = { nameInput } placeholder = { addressInput === undefined || addressInput === '' ? 'Name of the address' : addressInput }
-									onInput = { e => param.setNameInput((e.target as HTMLInputElement).value) }
-									maxLength = { 42 }
-								/>
-								<input disabled = { !param.addingNewAddress } className = 'input' type = 'text' value = { addressInput } placeholder = { '0x0...' }
-									onInput = { e => param.setAddressInput((e.target as HTMLInputElement).value) }
-									style = { `${ addressInput === undefined || ethers.utils.isAddress(addressInput.trim()) ? '' : 'color: var(--negative-color);' }` } />
-								<label class = 'form-control'>
-									<input type = 'checkbox' checked = { !askForAddressAccess } onInput = { e => { if (e.target instanceof HTMLInputElement && e.target !== null) { setAskForAddressAccess(!e.target.checked) } } } />
-									Don't request for an access (unsecure)
-								</label>
+								<div class = 'media-content' style = 'overflow-y: unset; overflow-x: unset;'>
+									<input className = 'input' type = 'text' value = { nameInput } placeholder = { addressInput === undefined || addressInput === '' ? 'Name of the address' : addressInput }
+										onInput = { e => setNameInput((e.target as HTMLInputElement).value) }
+										maxLength = { 42 }
+									/>
```

Comment:
Style: I'm not a fan of the arbitrary newlines here.  I would prefer all one line, or each property on a separate line (mild preference to all in one line)..
```suggestion
									<input className = 'input' type = 'text' value = { nameInput } placeholder = { addressInput === undefined || addressInput === '' ? 'Name of the address' : addressInput } onInput = { e => setNameInput((e.target as HTMLInputElement).value) } maxLength = { 42 } />
```
```suggestion
									<input
										className = 'input'
										type = 'text'
										value = { nameInput }
										placeholder = { addressInput === undefined || addressInput === '' ? 'Name of the address' : addressInput }
										onInput = { e => setNameInput((e.target as HTMLInputElement).value) }
										maxLength = { 42 }
									/>
```